core/src/navigation/LeftNav.html

```diff
@@ -200,16 +213,16 @@
                 class="fd-nested-list__icon {isSemiCollapsed ? 'sap-icon--rhombus-milestone-2' : ''}"
               ></span>
               {/if}
-              <span
-                class="fd-nested-list__title"
-              >{$getTranslation(node.label)}</span>
+              <span class="fd-nested-list__title">
+                <p>{$getTranslation(node.label)}</p>
```

Comment:
I thought we decided to not use the tag `<p>` since the solution I found should work without extra custom structure.